pom.xml

```diff
@@ -469,6 +473,13 @@
             <optional>true</optional>
         </dependency>
 
+        <!-- TOTP client -->
+        <dependency>
+            <groupId>com.warrenstrange</groupId>
+            <artifactId>googleauth</artifactId>
+            <version>1.1.2</version>
+        </dependency>
```

Comment:
An `<optional>true</optional>` would be great here that plugins which depend on AuthMe don't resolve this dependency.

Reply:
Of course! I just lazily added it to the POM until now to focus on the code impl (there are also no shading relocation rules yet). Keen eye ;)